active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
Great, thanks for the discussion on this, sounds like this type of metadata extends beyond `Sequence` objects.

> I would prefer interval_metadata over feature_metadata, because it is more specific and self-evident to have interval in the name and corresponding well to positional_metadata.

That's a good point, it does mirror `positional_metadata` nicely. I guess my problem with the name `interval_metadata` is that it doesn't include "feature" in the name. This type of metadata stores *features* associated with an object, where each *feature* exists over one or more *intervals*. The central piece of data that users will be interacting with is *features*, not *intervals* (they can query by feature property or location). Since the concept of a `Feature` is a fundamental part of this metadata, why not acknowledge that in the name? From other discussion in the RFC it sounds like `Feature` objects will be storing the intervals they exist on, so this object relationship is only becoming more solidified in the design ("an object has one or more *features*, and each *feature* exists on one or more *intervals*").